MonoGame.Framework/Random.cs

```diff
@@ -0,0 +1,386 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Xna.Framework
+{
+    public static partial class MathHelper
+    {
+        /// <summary>
+        /// Class for generating random numbers consistently when using Mono or Microsoft compilers.
```

Comment:
We need a much better description of this class.

For example there is no mention of LCG or link to information about what that is.  This is important as the API assumes you know how LCG works.

Also don't mention specific compilers...  just let people know that the results are consistent across platforms when using the same inputs.